docs/_docs/installation.md

```diff
@@ -37,11 +46,34 @@ Before you start, make sure your system has the following:
   </p>
 </div>
 
+## Install & Use Jekyll with Bundler
+
+To use Jekyll with [Bundler](https://bundler.io/), add it as a
+dependency of your project. It can be installed in a sub-directory of the
+project. This avoids permissions issues, and also allows you to use different
+versions of Jekyll with different projects.
+
+First, install Bundler (as a system-wide gem):
+```sh
+sudo gem install bundler
+```
+
+Then, add Jekyll to your project's dependencies:
+
+```sh
+cd my-jekyll-project
+bundle init                          # Creates an empty Gemfile
+bundle install --path vendor/bundle  # Set install path to ./vendor/bundle
+bundle add jekyll                    # Add & install Jekyll gem
+```
+
```

Comment:
I'm not a fan of this method though..
A new user would get confused by having to run `*bundle exec jekyll new .` and be greeted by an error message.,. while elsewhere in the docs, we simply instruct users to initialize a jekyll project with `jekyll new [jekyll-project-path]`

Reply:
We  could distinct new with defaults from blank (working on a PR to have a working default directory structure. with blank)